checkInput.ts

```diff
@@ -0,0 +1,35 @@
+let isValid: boolean[] = []
+
+for (let messageIndex = 0; messageIndex < numberOfMessages; messageIndex++) {
```

Comment:
There's often a support function for initing an array with a given value - could you check if there is one in TS?